mypyc/genops.py

```diff
@@ -1448,6 +1450,30 @@ def visit_raise_stmt(self, s: RaiseStmt) -> Value:
         self.add(Unreachable())
         return INVALID_VALUE
 
+    def visit_try_stmt(self, t: TryStmt) -> Value:
+        assert len(t.handlers) == 1 and t.types[0] is None and t.vars[0] is None, (
+            "Only bare except supported")
+        assert not t.else_body, "try/else not implemented"
+        assert not t.finally_body, "try/finally not implemented"
+
+        except_entry, exit_block = BasicBlock(), BasicBlock()
```

Comment:
This method is very clean after your basic block refactoring! Refactoring really can pay off quickly.